home-task1.2_Stream.js

```diff
@@ -1,9 +1,9 @@
-const fs = require('fs');
-const csv = require('csvtojson');
+import { createReadStream, createWriteStream } from 'fs';
+import csv from 'csvtojson';
 
 const CSV_FILE_PATH = 'csv/nodejs-hw1-ex1.csv';
 const OUTPUT_FILE_NAME = 'output.txt';
 
-const readStream = fs.createReadStream(CSV_FILE_PATH).setEncoding('utf-8');
-const writeStream = fs.createWriteStream(OUTPUT_FILE_NAME);
+const readStream = createReadStream(CSV_FILE_PATH).setEncoding('utf-8');
+const writeStream = createWriteStream(OUTPUT_FILE_NAME);
 readStream.pipe(csv()).pipe(writeStream);
```

Comment:
you also have to use pipeline to prevent loading contents of file fully into RAM.

Reply:
Sorry this is fine . My bad. Partial loading is okay for the task. There are ways where we can save more memory.